docs/02-Install.md

```diff
@@ -108,22 +108,23 @@ the [LORIS repository: Imaging Uploader Specification](https://github.com/aces/L
 
 2. **DICOM Archive**
 
-This LORIS module provides a front-end display of the details of the archived 
-DICOMs from the database `tarchive_*` tables. The only setting that impacts the 
-display of this module are the regex settings in the `Configuration` module 
-under the section `Imaging Modules`. These settings determine whether the 
+This LORIS module provides a front-end display with the details of the archived 
+DICOM study from the database `tarchive_*` tables. The only setting that 
+impacts the display of this module are the regex settings in the `Configuration` 
+module under the section `Imaging Modules`. These settings determine whether the 
 Patient Name/Patient ID header values are displayed in full, or show up as 
 **INVALID-HIDDEN**.
 
 More detailed specifications can be consulted in the 
-[LORIS repository: DICOM Archive Specification](https://github.com/aces/Loris/blob/master/modules/dicom_arhive/README.md).
+[LORIS repository: DICOM Archive Specification](https://github.com/aces/Loris/blob/master/modules/dicom_archive/README.md).
 
 
 3. **Imaging Browser**
 
-Imaging Browser accesses the PIC images directly from the filesystem where they
-are stored. It also provides the option to doownload some files.  Ensure that:
-- `/data/$PROJ` directory and subdirectories are readable and executable by
+The Imaging Browser module accesses the screenshot (PIC) images directly from 
+the filesystem where they are stored. It also provides the option to download 
+some files. Ensure that:
```

Comment:
I'm not sure if this is still relevant, but: "It also provides the option to download some files." => "By default, it also provides the option to download individual scans as a MINC file, or the entire session as a compressed file (_.tar_ extension)."

Reply:
I agree with you it is useful for the user, but the advantage of saying 'some files' is that we link to Imaging Browser where this detailed info is kept and more importantly, maintained up to date (as an example, Imaging browser should/will also allow to download niftii and other types of files, so makes sense to add this info in Imaging Browser when the feature is changed and not worry about tracking it here).

@um4r12 
do you agree or you still strongly feel we should add it here?

Reply:
@MounaSafiHarab You're right, don't think it's necessary to add it here, will just make it difficult to track changes in different locations. I had been reading from top to bottom, and I think it is mentioned somewhere further down anyways.